tox.ini

```diff
@@ -92,11 +93,17 @@ deps =
     aiopg012: aiopg>=0.12,<0.13
     aiopg013: aiopg>=0.13,<0.14
     aiopg: sqlalchemy
+    asyncpg014: asyncpg>=0.14,<0.15
     aiohttp12: aiohttp>=1.2,<1.3
+    aiohttp12: yarl==0.18.0
```

Comment:
in our master I think there is a fix for `yarl` so I think you can remove that

Reply:
removed